wherehows-backend/README.md

```diff
@@ -66,23 +62,18 @@ URL	 jdbc:mysql://<metastore host>:3306/metastore
 Set the hive metastore driver class to ```com.mysql.jdbc.Driver```
 other properties per configuration.
 
-Ensure these JAR files are present
-```
- lib/jython-standalone-2.7.0.jar
- lib/mysql-connector-java-5.1.36.jar
-```
 
 ### Run
 To run the backend service:
 
 Set the variables in application.env to configure the application.
 
-To Run backend service application on port 19001 (from the wherehows-backend folder):
+To Run backend service application on port 9001 (from the wherehows-backend folder):
```

Comment:
By default it's running on 19001: https://github.com/linkedin/WhereHows/blob/master/wherehows-backend/build.gradle#L4